bin/capsule-certs-generate

```diff
@@ -31,6 +31,7 @@ CONFIG_CONTENT = <<EOF
 :answer_file: #{answers_file.path}
 :installer_dir: #{INSTALLER_DIR}
 :installer_name: 'foreman-installer'
+:product_name: 'Katello'
```

Comment:
Scenario name is not IMO suitable for this, because if we need to refer to Katello server from Capsule scenario the name will be Capsule. I'm not very happy with 'product_name' too. Maybe `programm_name` or `server_name` would be better. Also moving the value to the `custom:` section could help.